CHANGELOG-2.x.md

```diff
@@ -2,6 +2,10 @@
 This changelog references the relevant changes done in 2.x versions.
 
 
+## v2.4.2
+* In `Triniti\Dam\AssetUploader::uploadToS3` add ability to upload to s3 ingest bucket.
```

Comment:
@bhsiao55 nit: i'd revise this to be more generic, something about allowing to customize bucket/key/acl via options